src/interfaces/store-interfaces/state-interfaces/business-information-interface.ts

```diff
@@ -11,6 +11,7 @@ export interface BusinessInformationIF {
   legalType: CorpTypeCd
   nrNumber?: string
   startDate?: string // YYYY-MM-DD
+  taxId?: string // may br BN9 or BN15
```

Comment:
This wasn't even in the UI's "schema" (or used by the UI). It's just one of those fields that the API provided and that the UI was sending back.

I had to add it here so the `delete` below would work.